### PR TITLE
Feature - Deployment page (Pre-Deployment)

### PR DESCRIPTION
**Pull request by cedric-matheus**
- Add pages module to styleguide
- Create Deployment Page example
- Create Deployment Page structure
- Create Deployment Page styles
- Modularize Deployment Page
- Add Deployment to Pages module
- Add Deployment Page to Routes

### CODE REVIEW

File: src/Pages/Deployment/Deployment.style.less [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.deploymentPage {
+  min-width: 720px;
+  min-height: 480px;
```
Comment (schafferjrdev) [inline src/Pages/Deployment/Deployment.style.less:3]: Poderia ser um `height: 100vh;` ao invés de ser um min

Comment (fberanizo) [reply src/Pages/Deployment/Deployment.style.less:3]: @cedric-matheus ^

Comment (cedric-matheus) [reply src/Pages/Deployment/Deployment.style.less:3]: @fberanizo @schafferjrdev

Vou colocar 100%, para garantir que ele se adapte a qualquer pai.

**Review by fberanizo — APPROVED** [review]
👍

**Pull request merged**